src/components/AuthForm.js

```diff
@@ -218,32 +259,26 @@ export default class AuthForm extends Component {
 
   constructor(props: Object) {
     super(props);
+    const {settings: {authMethods}} = this.props;
+    const defaultAuth = authMethods[0];
```

Comment:
yes, since we make it configurable. And for example we could have a stack where only fxa is allowed